deploy/non-olm/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: oadp-operator
           # Replace this with the built image name
-          image: "quay.io/konveyor/oadp-operator:latest"
+          image: "quay.io/konveyor/oadp-operator:v0.1.4"
```

Comment:
I have configured autobuilds for this and I plan to branch to v0.1.4 with this PR merge prior to submitting this to community operators.

Reply:
Cool !